src/parser/SparqlParser.cpp

```diff
@@ -62,7 +64,8 @@ void SparqlParser::parseQuery(ParsedQuery* query, QueryType queryType) {
     }
     auto parseResult =
         sparqlParserHelpers::parseConstructTemplate(str, std::move(prefixes));
-    query->_clause = std::move(parseResult.resultOfParse_);
+    query->_clause = std::move(parseResult.resultOfParse_)
+                         .value_or(ad_utility::sparql_types::Triples{});
```

Comment:
If you want to, you can do this in a separate PR,
but this  whole blocks looks like
`query->_clause = parseWithAntlr(...).value_or(...)`